src/store/store.ts

```diff
@@ -1197,6 +1185,47 @@ export const useStore = defineStore('store', {
 
     getIsRestorationTypeCourtOrder (): boolean {
       return !!this.stateModel.restoration.courtOrder?.fileNumber
+    },
+
+    /** The special resolution object. */
+    getSpecialResolution (): SpecialResolutionIF {
+      return this.stateModel.specialResolution
+    },
+
+    /** The special resolution validity flags. */
+    getSpecialResolutionFormValid (): boolean {
+      return this.getValidationFlags.flagsCompanyInfo.isValidSpecialResolution &&
+        this.getValidationFlags.flagsCompanyInfo.isValidSpecialResolutionSignature
+    },
+
+    /** The company info page validity flags. */
+    getSpecialResolutionConfirmValid (): boolean {
+      return this.getFlagsReviewCertify.isValidSpecialResolutionConfirm
+    },
+
+    getRules (): RulesMemorandumIF {
```

Comment:
I hope we don't have anything else in the state named "rules", haha.

Maybe this and memorandum, below, could be renamed to be a bit more specific?

Reply:
`getSpecialResolutionRules`
`getSpecialResolutionMemorandum`

Reply:
I like it :)

Reply:
Thanks. Wordy, I know, but better.